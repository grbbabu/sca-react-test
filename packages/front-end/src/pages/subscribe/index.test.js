import React from 'react';
import renderer from 'react-test-renderer';
import Subscribe from './index';

it('renders correctly when there are no input to the form', () => {
    const tree = renderer.create(<Subscribe />).toJSON();
    expect(tree).toMatchSnapshot();
});
