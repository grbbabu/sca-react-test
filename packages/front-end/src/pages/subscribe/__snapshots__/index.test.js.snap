// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no input to the form 1`] = `
<div>
  <div
    className="wrapper header"
  >
    <h1>
      Subscribe
    </h1>
  </div>
  <form
    className="wrapper formBody"
  >
    <div
      className="inputSpacing"
    >
      <label
        htmlFor="firstName"
      >
        First Name
        <span
          className="required"
        >
           *
        </span>
        <input
          id="firstName"
          name="firstName"
          onChange={[Function]}
          required={true}
          rest={Object {}}
          type="text"
          value=""
        />
      </label>
    </div>
    <div
      className="inputSpacing"
    >
      <label
        htmlFor="lastName"
      >
        Last Name
        <span
          className="required"
        >
           *
        </span>
        <input
          id="lastName"
          name="lastName"
          onChange={[Function]}
          required={true}
          rest={Object {}}
          type="text"
          value=""
        />
      </label>
    </div>
    <div
      className="inputSpacing"
    >
      <label
        htmlFor="email"
      >
        Email
        <span
          className="required"
        >
           *
        </span>
        <input
          id="email"
          name="email"
          onChange={[Function]}
          required={true}
          rest={Object {}}
          type="email"
          value=""
        />
      </label>
    </div>
    <div
      className="inputSpacing"
    >
      <label
        htmlFor="mobilePhone"
      >
        Mobile Phone
        <input
          id="mobilePhone"
          name="mobilePhone"
          onChange={[Function]}
          required={false}
          rest={Object {}}
          type="tel"
          value=""
        />
      </label>
    </div>
    <button
      className="button"
      disabled={true}
      onClick={[Function]}
      type="submit"
    >
      Submit
    </button>
  </form>
</div>
`;
